_aggregations/metric/weighted-avg.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Weighted average
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 150
+---
+
+# Weighted average
+
+The `weighted_avg` aggregation calculates the weighted average of numeric values across documents. This is useful when you want to calculate an average, but weight some data points more heavily than others.
+
+## Weighted average calculation
+
+The weighted average is calculated as `(sum of value * weight) / (sum of weights)`.
+
+## Parameters
+
+When using the `weighted_avg` aggregation you must define the following parameters:
+
+- `value`: The field or script to get the numeric values to average
+- `weight`: The field or script to get the weight for each value
+
+Optionally, you can specify the following parameters:
+
+- `format`: A numeric format to apply to the output value
+- `value_type`: A type hint for the values if using scripts or unmapped fields
+
+For the value or weight, you can specify:
+
+- `field`: The document field to use
+- `missing`: A value or weight to use if the field is missing
+
+
+## Using the aggregation
```

Comment:
**Technical reviewer: Please confirm this example is relevant to an OpenSearch user. I tested the example using Dev Tools. If another example is more appropriate, please replace the draft example with your example. Thank you.**

Reply:
LGTM